### DISCUSSION TITLE
Update to Corsican locale [co]

### DISCUSSION THREAD

**Discussion by Patriccollu** [aclap-dev/video-downloadhelper#134] [Language & Translation]
Hello,

Please find attached the latest update of **Corsican** localization based on beta version 8.2.0.23a1.

[VDH Corsican messages 8.2.0.23a1.json](https://github.com/aclap-dev/video-downloadhelper/files/14317034/VDH.Corsican.messages.8.2.0.23a1.json)

For information, the previous Corsican updates are available in this [specific thread on old Google group](https://groups.google.com/g/video-downloadhelper-internationalization/c/kVNf-8NuTbg).

Best regards,
Patriccollu.

**Comment by paulrouget** [comment]
Thank you! I will add that to the upcoming version.

**Comment by Patriccollu** [comment]
Hello,

Please find attached an updated **Corsican** localization based on version 9.0.2.3a1 (beta) which includes new strings for v9.
[VDH Corsican messages 9.0.2.3a1+.json](https://github.com/user-attachments/files/15857815/VDH.Corsican.messages.9.0.2.3a1%2B.json)

Best regards,
Patriccollu.

**Comment by Patriccollu** [comment]
As indicated in discussion #148, _Pull Requests_ can be submitted to propose changes directly in Corsican localization file `co.toml`, rather than sending another type of file (`.json` or `.txt`) within the current discussion thread.
